annlabmed.csl

```diff
@@ -0,0 +1,351 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal">
+  <info>
+    <title>Annals of Laboratory Medicine</title>
+    <id>http://www.zotero.org/styles/annlabmed</id>
```

Comment:
this should be `http://www.zotero.org/styles/annals-of-laboratory-medicine` ; correspondingly change link and filename.